app/src/format.py

```diff
@@ -66,6 +66,13 @@ def _get_bem_documents_to_show(
     return documents
 
 
+import markdown
+
+
+def to_html(text: str) -> str:
+    return markdown.markdown(text.replace("- ", "\n- ", 1))
```

Comment:
`markdown` expects `\n` before the start of a list.

Reply:
IIUC this is for situations where the text literally starts with a list -- if that's right, we should check `text.startswith("- ")`, since `- ` is a plausible part of prose (as someone who uses dashes a lot, for example, in this sentence.)